core/src/main/java/org/elasticsearch/index/query/GeoDistanceQueryParser.java

```diff
@@ -48,6 +52,9 @@
 
     public static final String NAME = "geo_distance";
 
+    public static final ParseField OPTIMIZE_BBOX_FIELD = new ParseField("optimize_bbox", "optimizeBbox");
+    protected static final DeprecationLogger deprecationLogger = new DeprecationLogger(Loggers.getLogger(BaseGeoPointFieldMapper.class));
```

Comment:
++ I had this before but was unsure if this approach would issue only a warning or fail the query. I'll revert to this and commit.